tests/test_cli.py

```diff
@@ -126,8 +126,6 @@ def test_count_builder_mapping(mock_path, tmp_path):
                 "study_python_counts_valid",
                 "-s",
                 "./tests/test_data",
-                "--database",
-                "test",
```

Comment:
This and similar changes is because duckdb_args provides --database already (and in a file that's properly in tmp_path), and this extra --database was confusing me until I figured that out.